commcare/src/test/java/org/motechproject/commcare/util/DummyCommcareSchema.java

```diff
@@ -162,4 +162,39 @@
 
         return applicationsInConfig2;
     }
+
+    public static List<CommcareApplicationJson> getApplicationsForConfigThree() {
+        List<CommcareApplicationJson> applicationsInConfig3 = new ArrayList<>();
+
+        Map<String, String> formNames6 = new HashMap<>();
+        formNames6.put("en", "form6");
+
+        List<String> fields1 = new ArrayList<>();
+
+        fields1.add(CASE_FIELD1);
+        fields1.add(CASE_FIELD2);
+        fields1.add(CASE_FIELD3);
+
+        FormSchemaQuestionJson questionJson7 = new FormSchemaQuestionJson();
+        questionJson7.setQuestionLabel("Last visit");
+        questionJson7.setQuestionValue(FORM_QUESTION4);
+
+        FormSchemaJson formSchemaJson6 = new FormSchemaJson();
+        formSchemaJson6.setFormNames(formNames6);
+        formSchemaJson6.setQuestions(Collections.singletonList(questionJson7));
+        formSchemaJson6.setXmlns(null);
```

Comment:
The whole point of this test is this one line - we should expose it more. Either add a comment, or preferably change the variable name for formSchemaWithNullXmlns